libs/books/feature/src/lib/snack-bar/snack-bar.service.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { TestBed } from '@angular/core/testing';
+import { MatSnackBar, MatSnackBarModule } from '@angular/material/snack-bar';
+
+import { SnackBarService } from './snack-bar.service';
+
+describe('SnackBarService', () => {
+  let service: SnackBarService;
+  let snackBar: MatSnackBar;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [MatSnackBarModule],
+      providers: [MatSnackBar]
+    });
+    snackBar = TestBed.inject(MatSnackBar);
+    service = TestBed.inject(SnackBarService);
+  });
+
+  it('should be created', () => {
```

Comment:
This test is pretty useless. Either add actual tests or just delete the file.

Reply:
removing this snackbar service along with the component.